Client.xcodeproj/project.pbxproj

```diff
@@ -1083,6 +1092,24 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
+		0ACF620E22FDC0D300297576 /* CopyFiles */ = {
```

Comment:
Now that DataTests don't use host app. All framework has to be copied to the test bundle